WRF.md

```diff
@@ -0,0 +1,204 @@
+# WRF 4.2 Porting Guide(CentOS 7.6)
+## Introduction
+The Weather Research and Forecasting (WRF) Model can be used for fine-scale weather simulation and forecasting, which is one of the important application scenarios of high-performance computing (HPC).
+
+For more information about the WRF, visit the official WRF website.
+
+Programming language: C/Fortran
+
+Brief description: mesoscale weather forecasting model
+
+Open-source protocol: public domain
+
+##### Recommended Version
+The recommended version is WRF V4.2
+## Environment Requirements
+### Software Requirements
+| Item  | Version  |  Download Address |
+| ------------ | ------------ | ------------ |
+|  HDF5 |  1.10.6 | https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.6/src/  |
+|  PNETCDF |  1.12.1 |  https://parallel-netcdf.github.io/wiki/Download.html |
+|  NETCDF-C | 4.7.3  |  https://github.com/Unidata/netcdf-c/releases/tag/v4.7.3 |
+| NETCDF-F  | 4.5.2  | https://github.com/Unidata/netcdf-fortran/releases/tag/v4.5.2  |
+|  WRF | 4.2  | https://github.com/wrf-model/WRF/archive/refs/tags/v4.2.tar.gz  |
+### OS Requirements
+| Item  | Version  | How to Obtain  |
+| ------------ | ------------ | ------------ |
+|  CentOS | 7.6  |  https://www.centos.org/download/ |
+| Kernel  | 4.14.0-115  |  Included in the OS image. |
+## Configuring the Compilation Environment
+### Installing dependencies
+
+
+    yum install time -y
+    yum install curl* -y
```

Comment:
Here we install `curl` but later we use `wget`. We need to either install `wget` in addition or update the downloads below to use `curl`.